portfolio/src/main/webapp/script.js

```diff
@@ -42,10 +42,52 @@ function fadeOut(){
 }
 
 //From Week-3 Tutorial
-function getGreeting() {
-  console.log('Handling the response.');
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementsByClassName('greeting-cont')[0].innerText = quote;
+function getComments() {
+  //Get the number of comments to show per page
+  numComments = localStorage.getItem("num-of-comments");
+  fetch(`/comments?num-comments=${numComments}`).then(response => response.json()).then((comment) => {
+    container = document.getElementsByClassName('comments-section')[0];
+    container.innerHTML = "";
+    for (item in comment) {
+        container.appendChild(createListElement(comment[item]));
+    }
   });
+  document.getElementsByName('num-comments')[0].value = numComments;
 }
 
+/** Creates an <li> element containing text. */
+function createListElement(comment) {
+  const liElement = document.createElement('li');
+  liElement.className += "comment-list";
+  
+  fetch("/comment.html").then(response=>response.text()).then(data =>{
```

Comment:
nit: add space before brace

run the code formatter

Reply:
done